generator/src/test/java/com/scottlogic/deg/generator/cucumber/LessThan.feature

```diff
@@ -2,12 +2,332 @@ Feature: User can specify that a numeric value is lower than, but not equal to,
 
 Background:
      Given the generation strategy is full
-      And there is a field foo
-      And foo is of type "numeric"
-      And foo is anything but null
+       And there is a field foo
+       And foo is of type "numeric"
+       And foo is anything but null
 
+# alone
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires integer data
+     Given foo is less than 100
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires decimal data
+     Given foo is less than 100.1
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo           |
+       | -2147482624.0 |
+       | -2147482623.9 |
+       | -2147482623.8 |
+       | -2147482623.7 |
+       | -2147482623.6 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires negative numbers data
+     Given foo is less than 0
+       And the generator can generate at most 5 rows
+     Then the following data should be included in what is generated:
+       | foo |
+       | -1  |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires 0
+     Given foo is less than 1
+       And the generator can generate at most 5 rows
+     Then the following data should be included in what is generated:
+       | foo |
+       | 0   |
+
+#lessThan
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: lessThan run against a non contradicting lessThan should be successful (lessThan 2 AND lessThan 5)
+     Given foo is less than 2
+       And foo is less than 5
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+Scenario: lessThan run against a non contradicting not lessThan should be successful (lessThan 5 AND not lessThan 1)
+     Given foo is less than 5
+       And foo is anything but less than 1
+     Then the following data should be generated:
+       | foo |
+       | 1   |
```

Comment:
will 1 be created for a not less than 1?

Reply:
`not(lessThan 1)` &equiv; `>= 1`

Reply:
I think this output is correct, what do you think?